lib/ship_quiet_logistics/documents/shipment_order.rb

```diff
@@ -68,9 +68,9 @@ def ship_to_hash
           'Address1'   => shipment.ship_address.address1,
           'Address2'   => shipment.ship_address.address2,
           'City'       => shipment.ship_address.city,
-          'State'      => shipment.ship_address.state.name,
+          'State'      => shipment.ship_address.state_name,
```

Comment:
@DanielWright the gem does not work with spree objects, but some decorated shipment objects, to make it decoupled from spree. The gem expects shipments to be a certain way already, so i thought it would not really matter to expect this format ?